src/meltano/devcontainer-feature.json

```diff
@@ -12,6 +12,31 @@
                 "latest"
             ],
             "type": "string"
+        },
+        "pythonVersion": {
+            "default": "3.11",
+            "description": "Select the version of Python to use.",
+            "enum": [
+                "latest",
+                "os-provided",
+                "3.12",
+                "3.11",
+                "3.10",
+                "3.9",
+                "3.8"
+            ],
+            "type": "string"
```

Comment:
The default value should be set to `os-provided`, and then `INTERPRETER` variable have to be set to the empty string `INTERPRETER=""` in `install.sh` -> [pipx-package](https://github.com/devcontainers-extra/features/tree/main/src/pipx-package) will use python3 (and install it if it's not there yet)

Setting the specific Python version (`3.12`, `3.11`, ...) requires the interpreter to be present there before pipx-package installs `meltano` in `install.sh` (for example, for `pythonVersion = 3.12` Python 3.12 has to be installed via [python feature](https://github.com/devcontainers/features/tree/main/src/python) beforehand). And that requirement should be put into the `description`

I don't think that having `latest` makes sense here - Should it be the latest Python version in general? The latest Python version available in the distribution repository? Or the latest one supported by `meltano`?